### PR TITLE
Feature/contrast anesthetics sens

### PR DESCRIPTION
**Pull request by bruriah1999**
diverged into contrast sensitivity and anesthetics materials sensitivity bc it's not an & relation  
consider having another base class similar to OptionBasedParser with a set of sentences indicating whether the column value is true or false in a yes/no question
edge cases 
WhatsApp Image 2023-09-23 at 22 44 45
image

### CODE REVIEW

File: parser_columns/blood_thinners.py [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-class BloodThinners(BaseParser):
-    def parse_document(self, document_content: str) -> str:
-        for blood_thinner in BLOOD_THINNERS_KEYWORD_TYPES:
-            if (blood_thinner in document_content) or (len(find_similar_words(document_content, blood_thinner)) > 0):
-                return blood_thinner
-        if (BLOOD_THINNERS_KEYWORD in document_content) or (
-                len(find_similar_words(document_content, BLOOD_THINNERS_KEYWORD)) > 0):
-            return DOESNT_TAKE_BLOOD_THINNERS_MESSAGE
-        return MISSING_VALUE_STRING
+# from utils.find_similar_words import find_similar_words
```
Comment (Guy-Shapira) [inline parser_columns/blood_thinners.py:1]: maybe you ment to remove this file changes from this branch?

File: parser_columns/anesthetics_sensitivity.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ANESTHETICS_SENSITIVITY_IMPLIERS = ["יש רגישות לחומר ניגוד"]
+
+
+class AnestheticsSensitivity(BaseParser):
+    def __init__(self) -> None:
+        super().__init__()
+
+    def parse_document(self, document_content: str) -> str:
+        try:
+            for no_anesthetics_implying_sentence in NO_ANESTHETICS_SENSITIVITY_IMPLIERS:
```
Comment (Guy-Shapira) [inline parser_columns/anesthetics_sensitivity.py:16]: did you check for typos in ruth documents?

File: parser_columns/contrast_materials_sensitivity.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+CONTRAST_MATERIALS_SENSITIVITY_IMPLIERS = ["וכי יש רגישות לחומר ניגוד וחומרי הרדמה"]
+
+
+class ContrastMaterialsSensitivity(BaseParser):
+    def __init__(self) -> None:
+        super().__init__()
+
+    def parse_document(self, document_content: str) -> str:
+        try:
+            for no_implying_sentence in NO_CONTRAST_MATERIALS_SENSITIVITY_IMPLIERS:
```
Comment (Guy-Shapira) [inline parser_columns/contrast_materials_sensitivity.py:15]: did you check for typos in ruth documents?

File: parser_columns/blood_thinners.py (hunk 1 above)
Comment (bruriah1999) [reply parser_columns/blood_thinners.py:1]: not here anymore

**Pull request merged**